src/main/resources/templates/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    <form action="/create-user" method="post">
+        <input type="text" placeholder="Enter your username" name="loginName"/>
+        <input type="text" placeholder="Enter your password" name="password"/>
```

Comment:
For the future `type="password"` will not redisplay the text.  While the movies do this all the time you really don't want to be showing off the password while they type it.